src/altinn-app-frontend/src/services/AppLanguageApi.tsx

```diff
@@ -0,0 +1,15 @@
+import { appApi } from 'src/services/AppApi';
+import { IAppLanguage } from 'altinn-shared/types';
+
+export const appLanguageApi = appApi.injectEndpoints({
+  endpoints: (builder) => ({
+    getAppLanguage: builder.mutation<IAppLanguage[], void>({
```

Comment:
This does not have to be a mutation as we simply fetch data and never really mutates anything. Can simply be a query 🥳  See docs: https://redux-toolkit.js.org/rtk-query/usage/queries